OpenOversight/app/templates/link_delete.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 {% block content %}
-  <div class="container theme-showcase" role="main">
+  <div class="container theme-showcase py-5" role="main">
```

Comment:
🔬  Should we add `py-5` as an addition to all of the `container` classes? Are they always used together?

Reply:
Since it's just the outer-most container, this seems good to me as is!